src/vaults.lua

```diff
@@ -53,8 +55,18 @@ end
 --- @param currentTimestamp number The current timestamp
 --- @param vaultId string The vault id
 --- @param allowUnsafeAddresses boolean|nil Whether to allow unsafe addresses, since this results in funds eventually being sent to an invalid address
---- @return Vault The created vault
-function vaults.vaultedTransfer(from, recipient, qty, lockLengthMs, currentTimestamp, vaultId, allowUnsafeAddresses)
+--- @param revokable boolean|nil Whether the vault is revokable
+--- @return Vault -- The created vault
+function vaults.vaultedTransfer(
+	from,
+	recipient,
+	qty,
+	lockLengthMs,
+	currentTimestamp,
+	vaultId,
+	allowUnsafeAddresses,
+	revokable
+)
 	assert(utils.isValidAddress(recipient, allowUnsafeAddresses), "Invalid recipient")
 	assert(qty > 0, "Quantity must be greater than 0")
 	assert(recipient ~= from, "Cannot transfer to self")
```

Comment:
Just a head check - when we call this function we should make sure that both of these fields are normalized and/or normalize here to reduce the possibility of user error.

Reply:
assertAndSanitize util formats the addresses before getting to the handler